src/components/Books.js

```diff
@@ -0,0 +1,70 @@
+const Books = () => {
+  const Book = [
+    {
+      id: 1,
+      title: 'Marianna',
+      author: 'Entre etre ou ne pas etre, je suis',
+    },
+    {
+      id: 2,
+      title: 'Moi, Tituba, Sorciere',
+      author: 'Christianna Forbes',
+    },
+    {
+      id: 3,
+      title: 'Reveilles-moi',
+      author: 'Rayley Giynes',
+    },
+  ];
+  return (
+    <ul>
+      {Book.map((book) => (
+        <li key={book.id} className="book">
+          <div>
+            {' '}
+            <span className="title">
+              {book.title}
+              {' '}
+            </span>
+            {' '}
+            <br />
+            {' '}
+            {book.author}
+            <Buttons />
+          </div>
+          <div>
+            <div className="statistics" />
+            {/* <span className="percentage">100%</span> */}
+            100%
+          </div>
```

Comment:
- [ ] Kindly create a Book component and place this data into it. It should: -

* Take the book data as a prop and renders the title and the author
* Have a button for removing books